### PR TITLE
Enable Windows Buildkite verification on default image Windows…

### PR DESCRIPTION
**Pull request by jaymalasinha**
Signed-off-by: Jaymala Sinha <jsinha@chef.io>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: spec/integration/knife/chef_repo_path_spec.rb [hunk 1]
```diff
@@ -79,7 +79,8 @@
         EOM
       end
 
-      it "knife list --local -Rfp --chef-repo-path chef_r~1 / grabs chef_repo2 stuff", :windows_only do
+      # "Skipping for BK... As Windows 2019 has 8dot3name disabled by default"
+      it "knife list --local -Rfp --chef-repo-path chef_r~1 / grabs chef_repo2 stuff", :windows_only && :skip_buildkite do
```
Comment (lamont-granquist) [inline spec/integration/knife/chef_repo_path_spec.rb]: `:windows_only && :skip_buildkite` looks a bit funky here.  i suspect ruby just evaluates this to `true` all the time?

Comment (jaymalasinha) [reply spec/integration/knife/chef_repo_path_spec.rb]: Update as per comment

**Comment by jaymalasinha** [comment]
@lamont-granquist this PR in now ready for review/merge

File: .expeditor/verify.pipeline.yml [hunk 2]
```diff
@@ -199,23 +199,27 @@ steps:
...
 
 - label: "Functional Specs Windows :ruby: 2.6"
   commands:
-    - /workdir/scripts/bk_tests/bk_win_functional.ps1
+    - scripts/bk_tests/bk_win_functional.ps1
+  expeditor:
+    executor:
+      windows:
+        privileged: true
+        shell: ["powershell", "-Command"]
```
Comment (jeremiahsnapp) [inline .expeditor/verify.pipeline.yml]: i think you can get rid of `shell: ["powershell", "-Command"]`. that was only relevant when this was run in docker right? not relevant when running the job directly on the host.

**Pull request merged**